cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -147,6 +147,20 @@ def test_service_ansible_playbook_crud(ansible_repository):
     assert not cat_item.exists
 
 
+def test_service_ansible_playbook_taging(request, ansible_catalog_item):
+    """ Tests ansible_playbook tag addition, check added tag and removal
+
+    Steps:
+        1. Login as a admin
+        2. Add tag for ansible_playbook
+        3. Check added tag
+        4. Remove the given tag
+    """
+    ansible_catalog_item.add_tag('Department', 'Support')
+    assert ansible_catalog_item.get_tags() == ['Department', 'Support']
+    request.addfinalizer(lambda: ansible_catalog_item.remove_tag('Department', 'Support'))
```

Comment:
I don't think this should be done as a request finalizer, I think it should be part of the test. call remove_tag, and then assert get_tags() returns empty, or at least doesn't have `['Department', 'Support']`